src/components/dialog-test.js

```diff
@@ -317,6 +332,12 @@ describes.realWin('Dialog', {}, (env) => {
       expect(() => dialog.open()).to.throw('already opened');
     });
 
+    it('should throw if iframe already connected when opening in a container', async () => {
```

Comment:
nit: same here, this description would start with "throws if...." instead

Reply:
also, does this test need to be async? I don't see any awaits in the test body.

Reply:
True, removed async and updated description